nucleus/parent/pom.xml

```diff
@@ -70,6 +70,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <legal.doc.source>${maven.multiModuleProjectDirectory}</legal.doc.source>
```

Comment:
Are there any requirement for this property to work ?

Reply:
correct env setup... given that there exists https://issues.apache.org/jira/browse/MNG-5830.
I think using rootlocation looks better as it is not tight to from where or where the build started

Reply:
the only drawback is that it runs for every module. Or I can move it to the root pom if you prefer that

Reply:
This needs to run for every module, correct ? The updated version looks fine IMO.

Reply:
yes, every produced source and binary jar should contain these file